### PR TITLE
adding submission for topic 4

### PR DESCRIPTION
**Pull request by Suchit153**

### CODE REVIEW

**Comment by lreiher** [comment]
Thanks for opening this PR!

It looks like you have added your content as a submodule. I cannot access that submodule. I would also suggest to consider placing the files directly inside this repository without yet another submodule.

**Comment by Suchit153** [comment]
> Thanks for opening this PR!
> 
> It looks like you have added your content as a submodule. I cannot access that submodule. I would also suggest to consider placing the files directly inside this repository without yet another submodule.

Done

File: reports/04-Time-Series-Inverse-Perspective-Mapping/2023_batu_suchit/python_scripts/Untitled.ipynb
Comment (lreiher) [inline reports/04-Time-Series-Inverse-Perspective-Mapping/2023_batu_suchit/python_scripts/Untitled.ipynb:1]: It looks like you still have some temporary leftover scripts that can be deleted, e.g., this `Untitled.ipynb`. I suggest to take a look at all files once more and delete those that are not required.

File: reports/04-Time-Series-Inverse-Perspective-Mapping/2023_batu_suchit/example_outputs/segmented_image_files/right/example_scene_0.npz
Comment (lreiher) [inline reports/04-Time-Series-Inverse-Perspective-Mapping/2023_batu_suchit/example_outputs/segmented_image_files/right/example_scene_0.npz:1]: You have a lot of `.npz` files. Can you please track them in [Git LFS](https://git-lfs.com/)? As seen in the [`.gitattributes`](https://github.com/ika-rwth-aachen/acdc-research-projects/blob/main/.gitattributes), I have already pre-configured a lot of file extensions for Git LFS, but I have missed `.npz`.

You can start tracking those by running `git lfs track "*.npz"`. Afterwards, however, you should also try to check whether those files already committed need to be adjusted, cause they do not get tracked by Git LFS automatically, if they have been checked in before.

The easiest solution could be to re-fork the repository or switch to a different branch, first track the files in Git LFS, and only afterwards commit them.

File: reports/04-Time-Series-Inverse-Perspective-Mapping/2023_batu_suchit/report.ipynb
Comment (lreiher) [inline reports/04-Time-Series-Inverse-Perspective-Mapping/2023_batu_suchit/report.ipynb:1]: Please list your contribution in [this section of the README](https://github.com/ika-rwth-aachen/acdc-research-projects#finished-research-projects).

**Review by lreiher — CHANGES_REQUESTED** [review]

File: reports/04-Time-Series-Inverse-Perspective-Mapping/2023_batu_suchit/python_scripts/Untitled.ipynb
Comment (Suchit153) [reply reports/04-Time-Series-Inverse-Perspective-Mapping/2023_batu_suchit/python_scripts/Untitled.ipynb:1]: Sorry about that... I thought I had deleted that file, but had only done it in my local
I have gone through the rest as well, everything else seems to be in order

**Pull request closed without merging**